filter.go

```diff
@@ -243,6 +256,9 @@ type MirredAction struct {
 	ActionAttrs
 	MirredAction MirredAct
 	Ifindex      int
+
+	Statistics *ActionStatistic
+	Timestamp  *ActionTimestamp
```

Comment:
Have you ever considered putting it into `ActionAttrs`?

Reply:
U are right, these statistics are indeed part of the `ActionAttrs`, which already included fields like `Index`, `Refcnt`, `Bindcnt`, etc. 
I have modified this point~